components/Header/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Link from "next/link";
+import { LinkItem } from './styles';
+
+type HeaderItem = {
+    label: string;
+    href: string;
+}
+
+type Props = {
+    items: Array<HeaderItem>
+};
+
+const Header = ({ items }: Props) => {
+    return (
+        <header>
+            <nav>
+                {items.map((item) => {
+                    return (
+                        <Link href={item.href} key={item.href}>
```

Comment:
is item.href guaranteed to be unique?

Reply:
I believe it should be, because it's always going to return the specific endpoint for that item like /quiz or the home route, so I believe that it'll always be unique, in our case. I might be thinking of that the wrong way, however.

Reply:
Sounds good, just being sure, I'll resolve.